lib/purescript-cst/src/Language/PureScript/CST/Flatten.hs

```diff
@@ -284,7 +284,7 @@ flattenType = \case
   TypeWildcard _ a -> pure a
   TypeHole _ a -> pure $ nameTok a
   TypeString _ a _ -> pure a
-  TypeInt _ a _ -> pure a
+  TypeInt _ _ a _ -> pure a
```

Comment:
I think this should be
```diff
- TypeInt _ _ a _ -> pure a
+ TypeInt _ a b _ -> foldMap a <> pure b
```
or something like that.

Reply:
Looking at the [layout tests](https://github.com/purescript/purescript/tree/master/tests/purs/layout) can help confirm this.